frontend/src/components/OperationsTableCard/OperationsTableCard.tsx

```diff
@@ -50,12 +60,26 @@ const OperationsTableCard: FunctionComponent<OperationsTableCardProps> = (props)
   const [showMyQueries, setShowMyQueries] = useState(props.showMyQueries);
   const [searchQuery, setSearchQuery] = useState('');
   const [info, setInfo] = useState('');
+  const [dropDownValues, setDropDownValues] = useState<DropdownItemProps[]>([]);
   const onModalHide = () => setInfo('');
+  // const { sources } = useContext(SourcesContext);
+  const { sources } = useContext(SourcesContext);
 
   useEffect(() => {
     fetchQueries(showMyQueries);
   }, []);
 
+  useEffect(() => {
+    console.log(`OperationTableCard ${sources}`);
```

Comment:
??

Reply:
Removed console logs and updated dropdown to be next to text search input